core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -126,13 +128,16 @@ protected KitDefinition parseDefinition(Element el) throws InvalidXMLException {
       kits.add(this.parse(child));
     }
 
+    if (BukkitUtils.isSportPaper()) {
+      // This is only possible in SportPaper
+      kits.add(this.parseKnockbackReductionKit(el));
+    }
```

Comment:
Instead of guarding from the call site, it's better to guard within the method. (This applies to other things as well)

Reason being: if the method is called from other places, it makes it harder to misuse.